tests/gtest/avif_fuzztest_helpers.h

```diff
@@ -167,13 +168,30 @@ inline auto ArbitraryAvifAnim() {
   return fuzztest::OneOf(ArbitraryAvifAnim8b(), ArbitraryAvifAnim16b());
 }
 
-// TODO: Try StructOf<Metadata>(StructOf<uint32_t[3]>())?
+// Generates two signed fractions where the first one is smaller than or equal
+// to the second one.
+inline auto ArbitraryMinMaxSignedFraction() {
+  return fuzztest::FlatMap(
+      [](int32_t max_n, uint32_t max_d) {
+        return fuzztest::Map(
+            [max_n, max_d](int32_t min_n) {
+              // For simplicity, use the same denominator for both fractions.
+              // This does not cover all possible fractions but makes it easy
+              // to guarantee that the first fraction is smaller.
```

Comment:
Nit: Add "than or equal to the second one", or change "smaller" to "not bigger".